docs/api/browser-window.md

```diff
@@ -1842,16 +1842,17 @@ will remove the vibrancy effect on the window.
 Note that `appearance-based`, `light`, `dark`, `medium-light`, and `ultra-dark` have been
 deprecated and will be removed in an upcoming version of macOS.
 
-#### `win.setTrafficLightPosition(position)` _macOS_
```

Comment:
We should mark the previous API deprecated and add these as new API in the interim!